### DISCUSSION TITLE
Can I write to visor's log tab?

### DISCUSSION THREAD

**Discussion by NotLazy** [deta/cloud-docs#287] [Help]
console.log doesn't seem to output to visor's log tab. is there a way to print to visor's log?
<img width="984" alt="image" src="https://user-images.githubusercontent.com/12667538/174769225-166914c7-5086-48c1-89be-06a21f638090.png">

**Comment by jnsougata** [comment]
The `Logs` only shows the value returned from the function so in order to print something in visor log you have to return the value from the function in string form

**Comment by abdelhai** [comment]
i believe log statements have to be inside functions/classes to be printed out in the visor
